Gordon360/Extensions/DateTimeExtensions.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+namespace Gordon360.Extensions.System;
+
+static public class DateTimeExtensions
+{
+    /// <summary>
+    /// Helper method that casts given date time to type UTC without changing the time
```

Comment:
I think this could be clearer:
```suggestion
    /// Specify the given DateTime as UTC, without changing the time value.
```